### PR TITLE
Add SonyMisc1 (Tag 0x9403) makernote tags

### LINKED ISSUE #280: Please add support for reading temperature values form RAW files
Hello,

my camera (Sony ILCE-7M3) has temperature information stored in the exif data. Especially the CameraTemperature value is interesting for [dark frame subtraction](https://en.wikipedia.org/wiki/Dark-frame_subtraction). You can create a dark frame library for subtraction if you know the temperature value and reuse them (+/- 1°C).

```
exiftool -s _7M30305.ARW | grep Temp
ColorTemperature                : Auto
CameraTemperature               : 8 C
BatteryTemperature              : 10.0 C
```

```
exiv2 -pa _7M30305.ARW | grep Temp
Exif.Sony2.ColorTemperature                  Long        1  0
```

RAW example can be found on https://raw.pixls.us/

I guess that other camera manufactures have also temperature sensors in the camera.

### PR DESCRIPTION
**Pull request by postscript-dev**
Source: Exiftool (https://exiftool.org/TagNames/Sony.html)

Closes #280.

### CODE REVIEW

File: src/sonymn_int.cpp [hunk 1]
```diff
@@ -806,6 +806,31 @@ namespace Exiv2 {
...
+
+    const TagInfo* SonyMakerNote::tagListSonyMisc1()
+    {
+        return tagInfoSonyMisc1_;
+    }
+
+    std::ostream& SonyMakerNote::printTemperatureInDegC(std::ostream& os, const Value& value, const ExifData*)
+    {
+        if (value.count() != 1)       os << "(" << value << ")";
+        else                          os << value << " °C";
```
Comment (kevinbackhouse) [inline src/sonymn_int.cpp:828]: This indentation is a bit weird. Please could you make it consistent with the rest of the file?

File: src/tags_int.hpp [hunk 2]
```diff
@@ -174,6 +174,7 @@ namespace Exiv2 {
         sony2CsId,
         sony2Cs2Id,
         sony2FpId,
+		sonyMisc1Id,
```
Comment (kevinbackhouse) [inline src/tags_int.hpp:177]: Please could you fix this indentation?

File: src/sonymn_int.cpp (hunk 1 above)
Comment (postscript-dev) [reply src/sonymn_int.cpp]: Thanks for looking at this. I have made the change and will use the usual format in future.

File: src/tags_int.hpp (hunk 2 above)
Comment (postscript-dev) [reply src/tags_int.hpp]: I was caught out by Eclipse's formatting the code. I have made the change and altered Eclipse's preferences.

**Review by kevinbackhouse — APPROVED** [review]

**Pull request merged**